runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -25,10 +25,6 @@
  */
 package net.runelite.client.plugins.slayer;
 
-import com.google.common.collect.ImmutableList;
```

Comment:
Why are the imports moved around?

Reply:
Whoops my auto-format apparently was set up to do this. Reverted to how it was before + changed my settings.